internal/gengapic/testdata/doc_file.want

```diff
@@ -21,6 +21,41 @@
 // (at https://api.foo.com) with Buz (at https://api.buz.com) and Baz (at
 // https://api.baz.com) to acclerate bar.
 //
+// Example Usage
```

Comment:
I believe the cloud.google.com style is to use sentence case for headings. So, `Client` and `Context` would be capitalized, but this should be `Example usage`.

Reply:
Thank you and updated! Good to know for the future as well.